saskatoon/member/forms.py

```diff
@@ -209,3 +212,21 @@ def save(self):
         instance.save()
 
         return instance
+
+
+class PasswordChangeForm(auth_forms.PasswordChangeForm):
+
+    def __init__(self, user, *args, **kwargs):
+        super().__init__(user, *args, **kwargs)
+
+        # replace widgets with placeholder values to fit Notika theme
+        self.fields['old_password'].widget = PasswordInput(attrs={'class': 'form-control', 'autocomplete': 'current-password', 'placeholder': 'Old password', 'autofocus': True })
+        self.fields['new_password1'].widget = PasswordInput(attrs={'class': 'form-control', 'autocomplete': 'new-password', 'placeholder': 'New password'})
+        self.fields['new_password2'].widget = PasswordInput(attrs={'class': 'form-control', 'autocomplete': 'new-password', 'placeholder': 'Confirm password'})
+
+    def save(self, commit=True):
+        instance = super().save(commit)
+        auth_user = AuthUser.objects.get(person=instance.person)
+        auth_user.password_set = True
+        auth_user.save()
```

Comment:
I think this should work: 
```suggestion
        person.auth_user.password_set = True
        person.auth_user.save()
```

Reply:
nice. even simpler - i used `instance.password_set`

Reply:
ok I didn't realize `instance` was of type `AuthUser`,  in that case I would prefer:
```
 def save(self, commit=True):
        self.instance.password_set = True
        super().save(commit)
```

Reply:
`self` is of type `PasswordChangeForm` though. i think AuthUser `instance` is returned from `save()`, hence the initial `save()` call.

Reply:
Oh okay true `self.instance` might be limited to `ModelForm` which `PasswordChangeForm` doesn't inherit from. fixed my suggestion